### DISCUSSION TITLE
Checkpoints

### DISCUSSION THREAD

**Discussion by Abelitux** [SIGGgames/PacoMonkey#24] [Ideas]
It's important to decide the concept of the checkpoints in the game. How and where does the player reappear when he dies?
In a game like ours (open-world), there are not that many options. One quite interesting is the Hollow Knight checkpoint system. Team Cherry uses benches, where you can relax and save the game. Our game should have this type of checkpoints (trees or smth), so that the player can play safely. Anyways, in Hollow Knight there are just a few benches, we should put more checkpoints because we don't want a frustrating game, don't we?
> ![hollow knight checkpoint](https://github.com/SIGGgames/PacoMonkey/assets/22718557/d00ded91-dbba-476e-8a02-39e895b5b846)

**Comment by Aniol0012** [comment]
I could be a big tree and this tree, depending on the biome, could change. This is an idea:

![big_tree_concept](https://github.com/SIGGgames/PacoMonkey/assets/53788631/1950982a-5c36-415b-af7b-66da9977171d)

**Comment by julianogales** [comment]
I like the idea of having just a few checkpoints, not making it too easy for the player since the game is not all about fighting and surviving.
If the player wants it easier or harder, one thing that could vary is the number of checkpoints, placing more if it's an easy experience and less if it's harder.
The mechanics of checkpoints and death could be that when the player dies, it would reappear on the last checkpoint like he had never moved from there, which means that all the enemies, bounties, dialogues and experience that the player gained from the checkpoint would reset again.
I added the Japanese Monkeys to the list of monkey species because they are known for taking baths in [thermal springs](https://www.google.com/search?sca_esv=586607062&sxsrf=AM9HkKl1gPO20j9XuKuhyN70CdTwkJApzQ:1701357243997&q=japanese+monkey+thermal+spring&tbm=isch&source=lnms&sa=X&ved=2ahUKEwikgKjRgeyCAxVeTKQEHUAXAPEQ0pQJegQIChAB&biw=1528&bih=790&dpr=1.25). We could use that as the checkpoint and placing different thermal springs around the map for the player to recover and chill.
